### ISSUE TITLE
What is the description of the data structure?

### ISSUE THREAD

**Issue by stephmather** [NVlabs/Dancing2Music#7]
I have extracted data.zip but I don't understand the data structure.  
What is the difference between the ballet_unit, ballet_unitseq3 and ballet_unitseq4 lists?
Are you able to share the original video sources rather than the numpy arrays?

**Comment by avbiswas** [comment]
I have the same question...

**Comment by HsinYingLee** [comment]
X_unit is for single dancing unit.
X_unitseq3 and X_unitseq4 are for sequences of 3 and 4 dancing unit, respectively.

**Comment by aryanna384** [comment]
Hi, @HsinYingLee still have several questions about the data structure:
1:demo.py can work well using the pre-trained model, but when I tried the train_decomp.py I met with:
`[Error 2]No such file or directory  #:'../data\zumba/zfwOSEZ8CkQ/10/unit/1.npy'
`
I checked the data/unitList/zumba_unit.txt, none of the files in '../data\zumba' can be found in zumba_unit.txt. 
Should I run another file before the train_decomp.py?

2.And what's the meaning of folders in zumba/x? The numerical order is different and discontinuous.

3.What's the exact meaning of:
> X_unit is for single dancing unit.
> X_unitseq3 and X_unitseq4 are for sequences of 3 and 4 dancing unit, respectively.

4.Did the data were processed into the dance unit already？What is the standard of segmentation and there is code in the demo? I want to know that if I have a series of motion data, how should I process to use in this project. 

Thanks so so so much!!!
